regolith/helpers/u_todohelper.py

```diff
@@ -101,21 +105,35 @@ def db_updater(self):
             return
         index = 1
         for t in todolist:
-            t["index"] = index
-            index += 1
+            if t.get('status') != "finished":
```

Comment:
please can you catch 'cancelled' as well

also, to make the code more readable, can you do "for todo in todolist:" etc. throughout the code.